src/locales/ja.yaml

```diff
@@ -79,17 +81,22 @@ app:
       not_found: ファイルが見つかりません
       processing: プロセッシング
     overlay:
-      label: <strong>ドラッグ</strong> ファイルをここに置く
+      drag_files_enqueue: <strong></strong> ファイルをここにドラッグしてキューに入れる。
+      drag_files_folders_upload: <strong></strong> 個のファイルとフォルダをここにドラッグしてアップロードします
```

Comment:
en.yaml:
> drag_files_folders_upload: <strong>Drag</strong> files and folders here to upload
> drag_files_enqueue: <strong>Drag</strong> files here to enqueue

The `<strong>` and `</strong>` delimiters here are to make the "Drag" word bold